core/templates/pages/admin-page/services/admin-router.service.ts

```diff
@@ -49,36 +58,36 @@ export class AdminRouterService {
   }
 
   /**
-   * Returns whether the activities tab is open.
+   * @returns {boolean} Returns whether the activities tab is open.
```

Comment:
```suggestion
   * @returns {boolean} Whether the activities tab is open.
```
The description should be about the returned value.